### ISSUE TITLE
Assert: Global paths are only supported for legacy databases

### ISSUE THREAD

**Issue by huguesv** [Microsoft/PTVS#3518]
If I turn OFF new style IntelliSense option, I get this assert for every IronPython and Conda environment that is discovered.

```
---------------------------
Assertion Failed: Abort=Quit, Retry=Debug, Ignore=Continue
---------------------------
Global paths are only supported for legacy databases



   at Microsoft.PythonTools.Interpreter.DatabasePathSelector.CalculateGlobalDatabasePath(InterpreterConfiguration config, Int32 version) in C:\Projects\GitHub\huguesv\PTVS\Python\Product\VSInterpreters\Interpreter\DatabasePathSelector.cs:line 32

   at Microsoft.IronPythonTools.Interpreter.IronPythonInterpreterFactoryProvider.GetCreationOptions(IServiceProvider site, InterpreterConfiguration config, Boolean& noDatabase) in C:\Projects\GitHub\huguesv\PTVS\Python\Product\IronPython\Interpreter\IronPythonInterpreterFactoryProvider.cs:line 258

   at Microsoft.IronPythonTools.Interpreter.IronPythonInterpreterFactoryProvider.EnsureInterpreter() in C:\Projects\GitHub\huguesv\PTVS\Python\Product\IronPython\Interpreter\IronPythonInterpreterFactoryProvider.cs:line 178

   at Microsoft.IronPythonTools.Interpreter.IronPythonInterpreterFactoryProvider.GetInterpreterFactory(String id) in C:\Projects\GitHub\huguesv\PTVS\Python\Product\IronPython\Interpreter\IronPythonInterpreterFactoryProvider.cs:line 148

   at Microsoft.PythonTools.Interpreter.InterpreterRegistryService.FindInterpreter(String id) in C:\Projects\GitHub\huguesv\PTVS\Python\Product\VSInterpreters\Interpreter\InterpreterRegistryService.cs:line 65

   at Microsoft.PythonTools.EnvironmentsList.ToolWindow.<>c__DisplayClass45_1.<UpdateEnvironments>b__3(InterpreterConfiguration c) in C:\Projects\GitHub\huguesv\PTVS\Python\Product\EnvironmentsList\ToolWindow.xaml.cs:line 394

   at Microsoft.PythonTools.Infrastructure.ObservableCollectionExtensions.Merge[TLeft,TRight,TKey](ObservableCollection`1 left, IEnumerable`1 right, Func`2 getLeftKey, Func`2 getRightKey, Func`2 project, IEqualityComparer`1 compareId, IComparer`1 compareSortKey) in C:\Projects\GitHub\huguesv\PTVS\Python\Product\Common\Infrastructure\ObservableCollectionExtensions.cs:line 105

   at Microsoft.PythonTools.EnvironmentsList.ToolWindow.UpdateEnvironments(String select) in C:\Projects\GitHub\huguesv\PTVS\Python\Product\EnvironmentsList\ToolWindow.xaml.cs:line 384

   at Microsoft.PythonTools.EnvironmentsList.ToolWindow.FirstUpdateEnvironments() in C:\Projects\GitHub\huguesv\PTVS\Python\Product\EnvironmentsList\ToolWindow.xaml.cs:line 433

   at System.Windows.Threading.DispatcherOperation.InvokeDelegateCore()

   at System.Windows.Threading.DispatcherOperation.InvokeImpl()

   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)

   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)

   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)

   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)

   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)

   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)

   at System.Windows.Threading.DispatcherOperation.Invoke()

   at System.Windows.Threading.Dispatcher.ProcessQueue()

   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)

   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)

   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)

   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)

   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)

   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)


---------------------------
Abort   Retry   Ignore   
---------------------------
```

**Comment by zooba** [comment]
Fairly trivial fix - we are passing the wrong version number from IronPython. I don't see why conda environments would trigger this though, and they aren't for me.

**Comment by zooba** [comment]
Fixed in 5066e37036cc8659a1aefd31e8afffea39399738
